docs/Design/Null Checking.md

```diff
@@ -0,0 +1,148 @@
+# Null Checking
+
+Creates checks for if the passed variable has value null. 
+
+A null variable is one that has no other value stored in it.
+
+Can be used to check whether the value is or is not null. 
+
+## Commands
+
+### `is null`
+
+`is null : variable`
+
+Creates a test that will return true if the variable is `null`.

```

Comment:
At this point, it's improper to refer to it as `null` with the code formatting. `null` indicates some specific code value.